### ISSUE TITLE
Event modal can't be closed through keyboard shortcut

### ISSUE THREAD

**Issue by abrassat** [xwikisas/application-mocca-calendar#225]
**Describe the bug**
Pressing 'esc' key to close the event modal doesn't work after the first time.

**Steps to reproduce**
Steps to reproduce the problematic behavior:
1. Go to MoccaCalendar home page
2. Create an event
3. Open the event modal by clicking on the event in the calendar
4. Try to press 'esc' key to close the modal

Alternative reproduction steps:
1. Go to MoccaCalendar home page
2. Click on an existing event to open the event modal
3. Press 'esc' key to close the modal
4. Click on an existing event to open the event modal
5. Try to press 'esc' key to close the modal

**Expected behavior**
The modal is closed

**Actual behavior**
The modal is not closed. JS Console output: `Uncaught TypeError: can't access property "removeChild", element.parentNode is null`

**Screenshots**

**Environment (please complete the following information):**
 - OS: linux mint
 - Browser firefox
 - XWiki version 15.10
 - App version 2.19.1

**Additional context**
